zerver/webhooks/github/view.py

```diff
@@ -91,10 +91,9 @@ def get_opened_or_update_pull_request_body(helper: Helper) -> str:
 def get_assigned_or_unassigned_pull_request_body(helper: Helper) -> str:
     payload = helper.payload
     include_title = helper.include_title
+    action = payload["action"].tame(check_string)
     pull_request = payload["pull_request"]
-    assignee = pull_request.get("assignee")
-    if assignee:
```

Comment:
Removed `assignee`, added `has_assignee` instead.
Added `action` variable to identify the operation.